modules/standard/Active/ActiveTest.enc

```diff
@@ -1,12 +1,13 @@
--- This file was automatically converted by encorec
-
 import Active.Active
```

Comment:
I would even argue that it *is* my responsibility, as it's PR that makes this library (verifiably) safe to use again.